src/js/boardView.js

```diff
@@ -0,0 +1,70 @@
+let boardSize;
+const pCellClsName = "innerCell";
+
+/* eslint-disable no-trailing-spaces */
+
+/*
+ Initializing entire game boards
+ TODO: score board that will ref to mainBoard vals ?
+*/
+export default function createGameBoards(brdSize){
```

Comment:
I assumed you're going to create classes for views that mirror the classes for boards, I'd like to talk F2F about this approach